src/__smoke-test__/smoke-test.spec.ts

```diff
@@ -0,0 +1,446 @@
+import { test, expect } from '@playwright/test';
+import {
+    HTTP_EVENT_TYPE,
+    JS_ERROR_EVENT_TYPE,
+    LCP_EVENT_TYPE,
+    FID_EVENT_TYPE,
+    PERFORMANCE_NAVIGATION_EVENT_TYPE,
+    PERFORMANCE_RESOURCE_EVENT_TYPE,
+    PAGE_VIEW_EVENT_TYPE,
+    SESSION_START_EVENT_TYPE
+} from '../plugins/utils/constant';
+
+// Secrete constant values retrieved from GitHub
+const endpoint = '${process.env.ENDPOINT}';
+const monitorId = '${process.env.MONITOR_ID}';
+const url = '${process.env.SMOKE_URL}';
```

Comment:
suggestion: Make these command line arguments instead of environment variables, since they are inputs to the test and do not need to be shared across processes.

Reply:
Did some research to find if Playwright accepts custom parameters from the command line, and the short answer is Playwright does not support this option.

Several feature requests were raised to support this, but the Playwright team stated that using env variables is the preferred option at the moment.

As a result, the proposed solution is the following:
1. define the npm command as `"cross-env URL=$URL MONITOR_ID=$MONITOR ENDPOINT=$ENDPOINT npx playwright test"` 
2. when we run `npm run smoke`, we pass in the parameters as `URL=test_url.com MONITOR=monitor_name ENDPOINT=endpoint  npm run smoke`